packages/editor/src/components/post-taxonomies/style.scss

```diff
@@ -10,30 +10,20 @@
 }
 
 .editor-post-taxonomies__hierarchical-terms-choice {
-	margin-bottom: 8px;
+	margin-bottom: $grid-unit-10;
+	&:last-child {
+		margin-bottom: $grid-unit-05;
+	}
 }
 
 .editor-post-taxonomies__hierarchical-terms-subchoices {
-	margin-top: 8px;
+	margin-top: $grid-unit-10;
```

Comment:
I saw both pixel values and variables used, so I updated everything to variables with the corresponding value while I was already here. 🙂